src/store.c

```diff
@@ -90,17 +91,17 @@ void write_generated_image_data_to_callback (struct context * context, const str
 void write_generated_image_data (void * restrict buffer, const struct data_node * data) {
   const struct data_node * node;
   for (node = data; node -> previous; node = node -> previous);
-  for (unsigned char * out = buffer; node; node = node -> next) {
-    memcpy(out, node -> data, node -> size);
-    out += node -> size;
+  for (unsigned char * output = buffer; node; node = node -> next) {
```

Comment:
The only other `out` variable is a `uint_fast32_t` in bmpwrite.c, not any pointers to output buffers.